src/images/icons/antrea.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This includes the word 'Antrea' in it, but we just want the icon.

Reply:
Plus one here

Reply:
I'll remove this file since it's on the spritesheet